tests/simulation/test_simulation.py

```diff
@@ -109,3 +126,203 @@ async def has_compact(node1, node2):
         await server3.start_client(PeerInfo(self_hostname, uint16(node1_port)))
         await server3.start_client(PeerInfo(self_hostname, uint16(node2_port)))
         await time_out_assert(600, node_height_at_least, True, node3, peak_height)
+
+    @pytest.mark.asyncio
+    @pytest.mark.parametrize(argnames="count", argvalues=[0, 1, 2, 5, 10])
```

Comment:
Mmm, so these are tests of the full node simulator rather than tests that the full node simulator is used for.  Yeah, I agree that's a good distinction and will look into creating another file.